Lukem/index page learn (#212)
moondance-labs/tanssi-docs: learn/tanssi/index.md

---
title: Learn About Tanssi

> description: Learn about Tanssi protocol and how it handles infrastructure complexities, making it easy for new appchains to be deployed in the Polkadot Web3 ecosystem.

Reviewer: ```suggestion
description: Learn about Tanssi protocol and how it handles infrastructure complexities, making it easy for new networks to be deployed in the Polkadot Web3 ecosystem.
```

icon: octicons-star-24
template: index-page.html
---

# Learn About Tanssi

> Here, you'll find an overview of Tanssi’s core capabilities that support appchain development, covering everything from infrastructure and security to essential integrations.

Reviewer: ```suggestion
Here, you'll find an overview of Tanssi’s core capabilities that support network development, covering everything from infrastructure and security to essential integrations.
```

> Key Features That Set Tanssi Apart:

Reviewer: ```suggestion
Key features that set Tanssi apart:
```

This isn't really a title, so we should treat it as a normal sentence that uses sentence case

> Here, you’ll find everything that sets Tanssi apart as a leading solution for appchain development. From block production as a service to flexible security providers and essential integrations, this section highlights the core features that make Tanssi a robust and adaptable protocol for building scalable, secure, and innovative appchains.

Reviewer: ```suggestion
Here, you'll find an overview of Tanssi’s core capabilities that support appchain development, covering everything from infrastructure and security to essential integrations.
```

Since the below list includes block production as a service, flexible security providers, etc. lets keep this first sentence short and sweet

> Key Features That Set Tanssi Apart:

Reviewer: Same comments here as on the other pages about list formatting

Reviewer: Nice bullet points here!

> - **Block production as a service** - tanssi’s incentivized block producers ensure reliable block creation and network liveness, removing the operational burden from developers

Reviewer: ```suggestion
- **Block production as a service** - Tanssi’s incentivized block producers ensure reliable block creation and network liveness, removing the operational burden from developers
```

- **Flexible security providers** - choose security solutions tailored to your appchain’s environment, such as Symbiotic for Ethereum-based chains, enabling shared security without sourcing economic guarantees
- **Time-to-market efficiency** - launch appchains in minutes with Tanssi’s comprehensive approach to infrastructure and security, significantly reducing development time
- **Seamless integrations** - Tanssi provides out-of-the-box infrastructure, including wallets, RPC endpoints, and block explorers, streamlining appchain deployment

## Explore What This Section Covers

> :::INSERT_GENERATED_CARDS::: 

Reviewer: I think we could switch up some of these icons for a more cohesive look (i.e., the width of the lines, whether or not elements are filled in, etc.). 

<img width="821" alt="Screenshot 2024-12-16 at 12 30 51 AM" src="https://github.com/user-attachments/assets/2564f2f1-0f5d-4990-9dbb-2178e3e60e22" />
